spec/support/active_record.rb

```diff
@@ -1,3 +1,5 @@
+require_relative 'environment'
```

Comment:
It's not that it's needed, but to be clear that `support/environment` is being required and not some other environment file from elsewhere.